amundsen_application/static/js/components/NavBar/tests/index.spec.tsx

```diff
@@ -40,8 +40,19 @@ describe('NavBar', () => {
     beforeEach(() => {
         props = {
           loggedInUser:  {
+            email: 'test@test.com',
```

Comment:
nit: You can consider putting this in `fixtures/globalState` so we can ruse it if we need `loggedInUser` test data anywhere else in the future.